### PR TITLE
(prometheus) minor fix of auto complete

### PR DESCRIPTION
**Pull request by mtanda**

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/mode-prometheus.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ var PrometheusHighlightRules = function() {
       regex : "[a-zA-Z_$][a-zA-Z0-9_$]*\\b"
     }, {
       token : "keyword.operator",
-      regex : "\\+|\\-|\\*|\\/|%|\\^|=|==|!=|<=|>=|<|>|=\\~|!\\~"
+      regex : "\\+|\\-|\\*|\\/|%|\\^|==|!=|<=|>=|<|>"
```
Comment (mtanda) [inline public/app/plugins/datasource/prometheus/mode-prometheus.js]: label match operator is added to another `keyword.operator` token. no need to add here.

File: public/app/plugins/datasource/prometheus/mode-prometheus.js [hunk 2]
```diff
@@ -75,8 +75,7 @@ var PrometheusHighlightRules = function() {
       regex : '"[^"]*"|\'[^\']*\''
     }, {
       token : "punctuation.operator",
-      regex : ",",
-      push  : 'start-label-matcher'
+      regex : ","
```
Comment (mtanda) [inline public/app/plugins/datasource/prometheus/mode-prometheus.js]: I'm not understand well, but the label match auto complete work without pushing `start-label-matcher`.

File: public/app/plugins/datasource/prometheus/mode-prometheus.js [hunk 3]
```diff
@@ -118,11 +117,6 @@ var prometheusFunctions = [
     docText: 'Returns the input vector with all sample values converted to their absolute value.'
   },
   {
-    name: 'abs()', value: 'abs',
```
Comment (mtanda) [inline public/app/plugins/datasource/prometheus/mode-prometheus.js]: The entry is duplicated. remove.

File: public/app/plugins/datasource/prometheus/mode-prometheus.js (hunk 2 above)
Comment (mtanda) [reply public/app/plugins/datasource/prometheus/mode-prometheus.js]: Yes, the `start-label-matcher` is referenced at https://github.com/grafana/grafana/pull/9555/files#diff-52972fb05f645321f656a0cc5f9c99d4R56 .

Comment (mtanda) [reply public/app/plugins/datasource/prometheus/mode-prometheus.js]: And the role of `keyword.operator` is not same.
One is for calculation, another one is for label match.

**Comment by bergquist** [comment]
:+1:

**Pull request merged**